### DISCUSSION TITLE
Windows Taskbar Icon duplicates

### DISCUSSION THREAD

**Discussion by achestaro** [seerge/g-helper#284] [Ideas]
Any plans to make the "Run on Startup" setting more compatible with the Windows taskbar?
I currently have 6 entries of "GHelper" in the Taskbar "Hidden icon menu". This might be due to the app being an independent executable every time there is a new version. The following screen capture only shows 3/6 entries since they do not fit in the screen.
![ghelpertaskbar](https://user-images.githubusercontent.com/11967190/233796747-bcdc53bd-7ea3-448a-809c-f32c335dccaf.jpg)

**Comment by seerge** [comment]
@achestaro this has nothing to do with being "compatible" with the taskbar. You just run the app from different locations so windows thinks it's different apps (i can imagine directly from zip file resulting exe being inside temp folder ?)

**Reply by achestaro** [reply]
That is correct, I create a new folder for each update in case I need to revert back. I guess there are no plans to make this an MSI that saves to Program Files? Thanks for the app.

**Reply by seerge** [reply]
There is no plans to add installer, idea to keep it as light and simple as possible. You can always revert back to any version by just downloading it from git, they are all under releases.
